templates/definition/meter/huawei-sun2000-dongle-powersensor.yaml

```diff
@@ -79,6 +79,16 @@ render: |
       address: 32064 # Input power DC (if no battery in your system - for more precise readings use 32080 # Active generation power AC)
       type: holding
       decode: int32
+  energy:
+    source: modbus
+    {{- include "modbus" . | indent 2 }}
+    model: huawei
+    timeout: 15s
```

Comment:
Sollte das nicht ein Parameter mit default sein?

Reply:
Gute Frage. Ich habe das so übernommen.
Das ist in allen Templates für das zickige SDongle so.

Reply:
Vorschlag ist drin